packages/cli/src/lib/onboarding/local.ts

```diff
@@ -170,7 +179,14 @@ export async function setupLocalStarterTemplate(
       );
     }
 
-    return Promise.all(promises).catch(abort);
+    return Promise.all(promises)
+      .then(() => {
+        if (storefrontToLink) {
+          // Save linked storefront in project
+          setStorefront(project.directory, storefrontToLink);
```

Comment:
Let's ensure this is also added to the promise chain, since it's an async operation:

```suggestion
          return setStorefront(project.directory, storefrontToLink);
```